src/lambda/local/debugConfiguration.ts

```diff
@@ -4,18 +4,17 @@
  */
 
 'use strict'
-
 import * as vscode from 'vscode'
 
-export interface NodeDebugConfiguration extends vscode.DebugConfiguration {
-    readonly type: 'node'
+// TODO: Figure out common attributes of a debug configuration that belong here
```

Comment:
The common attributes are those included in the `vscode.DebugConfiguration` interface. The rest should be assumed to be language-specific. So we'll want two separate sub-interfaces for node and python.

Reply:
Agreed. And currently the language specific interfaces are only used in their respective CodeLens provider so they need not be in in debugConfiguration.ts for now. Hence the `git mv` from [nodeDebugConfiguration.ts](https://github.com/aws/aws-toolkit-vscode/blob/develop/src/lambda/local/nodeDebugConfiguration.ts) to `debugConfiguration.ts`